global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "2.2.203",
+    "dotnet": "2.2.204",
```

Comment:
Is this change intentional?

Reply:
It brings it up to the current SDK version. I always install them as they are released and uninstall the previous ones, so I don't have 2.2.203 installed any more.

Reply:
Set global.json to the lowest that will not break a contributor, since setting this higher will force a contributor to upgrade their SDK. 

There are few cases where requiring a specific patch is needed. If the one specified is not found, any higher patch (last 2 digits) will be used. 

@jonsequitur are we using anything specific to 2.2? To 2.2.200? 

2.2 is particularly concerning since it goes out of support sooner and would force an upgrade to 3.0.  2.1.300 may be a better choice for System.CommandLine to correspond to the lowest needed for global tools.